go/kbfs/libkbfs/favorites.go

```diff
@@ -317,6 +321,10 @@ func favoriteToFolder(fav favorites.Folder, data favorites.Data) keybase1.Folder
 func (f *Favorites) handleReq(req *favReq) (err error) {
 	defer func() { f.closeReq(req, err) }()
 
+	if req.refresh {
+		<-f.refreshWaiting
+	}
```

Comment:
Do you still need a `req.refresh`? Seems like you  can just do:
```go
select {
case <-f.refreshWaiting:
	...
default:
}
```

Reply:
`req.refresh` indicates that this particular request is a refresh request. In order to prevent multiple refresh requests from working at the same time, I added this channel in addition to the wait group; however, I claim the boolean is not a sync construct at all.